src/rpm/rpm.go

```diff
@@ -0,0 +1,142 @@
+package rpm
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/tarantool/cartridge-cli/src/common"
+	"github.com/tarantool/cartridge-cli/src/project"
+)
+
+// Many thanks to @knazarov, who wrote packing in RPM in Lua a long time ago
+// This code can be found here
+// https://github.com/tarantool/cartridge-cli/blob/cafd75a5c8ddfdb93ef8290d6e4ebd5d83e4c46e/cartridge-cli.lua#L1814
+
+//  RPM file is a binary format, consisting of metadata in the form of
```

Comment:
`RPM file is a binary format`

File is not format.

RPM file in binary representation... Or smth similar...

Reply:
It seems to be copy-pasted from https://blog.packagecloud.io/eng/2014/10/20/yum-createrepo-generate-incorrect-metadata/

Reply:
fixed